ciri/conf.c

```diff
@@ -68,6 +68,17 @@ static sponge_type_t get_sponge_type(char const* const sponge_type) {
   return map[i].type;
 }
 
+static retcode_t get_true_false(char const* const input, bool* const output) {
+  if (strcmp(input, "true") == 0) {
+    *output = true;
+  } else if (strcmp(input, "false") == 0) {
+    *output = false;
+  } else {
+    return RC_CIRI_CONF_INVALID_ARGUMENTS;
```

Comment:
should we remain it to the default value or do we have a mechanism to locate which argument went wrong?

Reply:
It will be done in #583